study.md

```diff
@@ -134,15 +134,15 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a brief description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+HTTP stands for HyperText Transfer Protocol. This is process is the path upon which the front end (meaning your web browser) is connected to the actual webpage's server on the backend. To drastically simplify, the client requests the pages HTML from the server, which the server prepares and sends back to the client, which then processes it and displays it for the user.
 ```
 
 ## Describe what a client is and what a server is
 
  What is a client and what is a server? How do they interact with each other?
 
 ```md
-<!-- your answer here -->
+The client is the front-end, user guided program which makes requests for HTML, recieves said requests and renders them for the user. The server exists on the back end, recievies the requests and sends the HTML responses to the client. Server can refer to both the program recieving the request/sending the response as well as the physical machinery on which the program is run and the HTML is stored.
```

Comment:
Yup, but the client does not only exist to make requests for HTML. Remember, the client can also send data (such as login info), update data, and delete data.